pkg/updater/org_config.go

```diff
@@ -99,6 +117,8 @@ func (c *orgConfig) waitForCatalog(ctx context.Context) error {
 	select {
 	case <-ctx.Done():
 		return ctx.Err()
+	case <-c.catalogReceivedTimeout.Done():
+		return nil
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Shouldn't we log something here?